perfkitbenchmarker/linux_virtual_machine.py

```diff
@@ -90,6 +94,13 @@
                   '/sys/kernel/mm/transparent_hugepage/enabled before starting '
                   'the benchmark.')
 
+flags.DEFINE_bool('network_enable_BBR', False,
+                  'A short cut to enable BBR congestion control on the network.'
```

Comment:
shortcut

Reply:
need spaces before all of your closing quotes in the flag descriptions, otherwise it will look funny when printed to screen.